package.json

```diff
@@ -1,13 +1,18 @@
 {
-  "name": "aol-hb.js",
+  "name": "aol-hb",
   "version": "1.0.0",
   "description": "Header bidding library for AOL's demand side platform.",
-  "main": "src/aol-hb.js",
+  "main": "src/build/aol-hb.js",
```

Comment:
Common practice is to name the directory with compiled files `dist` (meaning distribution). But it's fine also as it is now, so feel free to decide.